src/__tests__/Module-test.js

```diff
@@ -77,6 +76,7 @@ describe('Module', () => {
     (options) => createModule({...options, file: '/root/package.json'});
 
   beforeEach(function(done) {
+    process.platform = 'linux';
```

Comment:
Hmm, this breaks the few places we need to cache the actual platform (used to verify path.basename, which is arch-relative, for example).

Let's look into this later on.